Dfu-programmer cannot cannot find libusb. This patch fixes that.

diff --git a/src/dfu-device.h b/src/dfu-device.h
index bbdfa99..31af0e8 100644
--- a/src/dfu-device.h
+++ b/src/dfu-device.h
@@ -6,7 +6,7 @@
 #endif
 #include <stdint.h>
 #ifdef HAVE_LIBUSB_1_0
-#include <libusb.h>
+#include <libusb-1.0/libusb.h>
 #else
 #include <usb.h>
 #endif

diff --git a/src/main.c b/src/main.c
index ef2aa16..dcc3636 100644
--- a/src/main.c
+++ b/src/main.c
@@ -24,7 +24,7 @@
 #include <stdio.h>
 #include <string.h>
 #ifdef HAVE_LIBUSB_1_0
-#include <libusb.h>
+#include <libusb-1.0/libusb.h>
 #else
 #include <usb.h>
 #endif

diff --git a/src/dfu.c b/src/dfu.c
index 7dbcc57..952e896 100644
--- a/src/dfu.c
+++ b/src/dfu.c
@@ -27,7 +27,7 @@
 #include <stdlib.h>
 #include <stddef.h>
 #ifdef HAVE_LIBUSB_1_0
-#include <libusb.h>
+#include <libusb-1.0/libusb.h>
 #else
 #include <usb.h>
 #endif

diff --git a/src/dfu.h b/src/dfu.h
index 2a159e8..63510f2 100644
--- a/src/dfu.h
+++ b/src/dfu.h
@@ -25,7 +25,7 @@
 # include <config.h>
 #endif
 #ifdef HAVE_LIBUSB_1_0
-#include <libusb.h>
+#include <libusb-1.0/libusb.h>
 #else
 #include <usb.h>
 #endif
